demo-dep-output.yaml

```diff
@@ -894,257 +894,233 @@
 - fileURI: file:///analyzer-lsp/examples/java/pom.xml
   provider: java
   dependencies:
-  - name: io.fabric8.kubernetes-client-api
-    version: 6.0.0
+  - name: com.fasterxml.jackson.core.jackson-core
```

Comment:
There seems to be a new order happening. I don't think that I see new entries here. Can you help me understand if this is intended, and does it break any other tests?